docs/source/services/timelock_service/manual-migration.rst

```diff
@@ -92,16 +95,21 @@ The steps for invalidating the old AtlasDB timestamp will vary, depending on you
 
         ALTER TABLE atlasdb_timestamp RENAME last_allocated TO LEGACY_last_allocated;
 
-- If using Cassandra, one method of invalidating the table is to overwrite the timestamp bound record with the
-  empty byte array (consider using ``cqlsh`` to do this). This table is stored in the same keyspace that your
+- If using Cassandra, one method of invalidating the table is to overwrite the timestamp bound record with an invalid
+  byte array. We recommend using a bogus one-byte array for this; the zero byte array is a deletion sentinel, and
+  if supplying byte arrays longer than 8 bytes, we will interpret the first 8 bytes as the timestamp bound.
+  Automated migration is implemented in this way as well (though we use Cassandra's lightweight transactions for
+  the automated migration, to be resilient to server lag when restarting an AtlasDB client's cluster with a Timelock
+  block for the first time).
+  This can be done easily using ``cqlsh``. The timestamp table is stored in the same keyspace that your
   AtlasDB client uses for its key-value service.
 
      .. code:: bash
 
         SELECT * FROM atlasdb."_timestamp";
         <note the value returned by this - call this K>
-        INSERT INTO atlasdb."_timestamp" (key, column1, column2, value) VALUES (0x7472, 0x7472, -1, K);
-        INSERT INTO atlasdb."_timestamp" (key, column1, column2, value) VALUES (0x7473, 0x7473, -1, 0x);
+        INSERT INTO atlasdb."_timestamp" (key, column1, column2, value) VALUES (0x7473, 0x7472, -1, K);
```

Comment:
To be fair I think when this was written I didn't even know that lightweight transactions require the same partition key. Now I do :) thanks!